apps/condo/domains/ticket/utils/tables.utils.ts

```diff
@@ -75,6 +75,22 @@ export const getPropertyScopeFilter = () => {
     }
 }
 
+export const getTicketTypeFilter = (favoriteTicketIds) => {
+    return function getWhereQuery (options) {
+        if (isEmpty(options) || !Array.isArray(options)) return
+
+        if (options.includes('common') && options.includes('favorite')) {
```

Comment:
this way it would be a bit more readable and skip extra calculations

```
const hasCommon = options.includes('common')
const hasFavourite = options.includes('favourite')

if (hasCommon && hasFavourite) return {}
if (hasCommon) return { id_not_in: favouriteTicketIds }
if (hasFavourite) return { id_in: favouriteTicketIds }
```

by the way please fix typo favorite -> favo**U**rite